app/services/db/piv_cac_configuration.rb

```diff
@@ -0,0 +1,18 @@
+module Db
+  class PivCacConfiguration
```

Comment:
were there any specs for the old separate versions of these classes that we can also combine into new specs? 🙏 😬

Reply:
There were not...

I'm mostly okay with it though since the methods are limited to a single ActiveRecord call each.

Reply:
😭 bummer